runelite-client/src/main/java/net/runelite/client/plugins/instancemap/InstanceMapInputListener.java

```diff
@@ -99,6 +99,11 @@ public MouseEvent mouseClicked(MouseEvent event)
 			return event;
 		}
 
+		if (isWithinCloseButton(event.getPoint()))
```

Comment:
You should maybe check for left-click button pressed (SwingUtil.isLeftMouseButton or something like that)